mk/aosp_optee.mk

```diff
@@ -106,7 +106,11 @@ include $(CLEAR_VARS)
 LOCAL_MODULE := $(local_module)
 LOCAL_PREBUILT_MODULE_FILE := $(OPTEE_TA_OUT_DIR)/$(LOCAL_MODULE)
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR)/lib/optee_armtz
+ifneq ($(filter %.ta, $(local_module)),)
```

Comment:
I'm trusting that you're doing the right thing here.
AOSP support is more or less unmaintained so if this works for you...

Reply:
I hope so. Since TAs and shared libraries recognized by tee-supplicant are with file extension ".ta", this should generally work for most cases. We did test by building optee_test suite.

Or maybe we can add an optional variable, `local_module_class`, for the module to specify its class. That could make it more straightforward.